### ISSUE TITLE
53450848 - Sonic Generations

### ISSUE THREAD

**Issue by Beta21Collector** [xenia-project/game-compatibility#391]
http://marketplace.xbox.com/en-US/Product/Sonic-Generations/66acd000-77fe-1000-9115-d80253450848
Tested on: EmuCR-Xenia-20160115
Crashes instantly.
Log: https://gist.github.com/Beta21Collector/3aae336bce54eaa4440b
state-crash

**Comment by Lumadash** [comment]
This is really needed to work. The PC version is very flawed.

**Comment by DrChat** [comment]
Reserves memory without MEM_LARGE_PAGES and then commits it with MEM_LARGE_PAGES using the unaligned address given from the reservation.

**Comment by DrChat** [comment]
Try this again - might be fixed as of 54ba3480e471cc9d2fb8833c043d83725ebee70e.

**Comment by shoopdahoop22** [comment]
I would test it... if Xenia wasn't so broken on AMD cards right now!

Xenia now crashes instantly on AMD cards because it thinks Vulkan is not installed.

**Comment by TotalCaesar659** [comment]
Tested on (https://github.com/benvanik/xenia/commit/abf162c0e32889b37461374b1d25ef9b5bba2648)

# Issues:
Instant crash.

# Log:
[Log file](https://drive.google.com/file/d/0B7KpdfS_wYnZN3pvaWhzTnQ1am8/view)

# Screenshot(s):
![0000](https://cloud.githubusercontent.com/assets/14265316/17650204/80bce6b4-6250-11e6-9e3e-021bd067bf56.jpg)

# Labels:
(state-intro)

**Comment by FitterSpace** [comment]
### Tested on https://github.com/xenia-project/xenia/commit/f748e5af49062fd78d098acdaf045183f29faf36

The guest crashes immediately upon loading the image.

### Log File (12 KB):
[SonicGenerations.zip](https://github.com/xenia-project/game-compatibility/files/3665933/SonicGenerations.zip)

**Comment by islander0** [comment]
### **Tested on** [xeniaproject/xenia@1d35ba9e](https://github.com/xenia-project/xenia/commit/1d35ba9ee7be6c57bb32f97721a05ae13cd4cb45#diff-a167d0527d7df721bfc73a30c1da2fce)
The game boots up and works fine up until a certain point in the introductory level.
![Title Screen](https://user-images.githubusercontent.com/25135838/85903611-79f95a80-b7e5-11ea-8840-dca7e397d057.png)

### **Issues**
The textures become glitchy for a small amount of time:
![SG - Glitchy Textures](https://user-images.githubusercontent.com/25135838/85903633-8d0c2a80-b7e5-11ea-8225-a6399f8937bb.png)

Some shaders aren't rendering:
![SG - Missing Shaders](https://user-images.githubusercontent.com/25135838/85903729-d52b4d00-b7e5-11ea-9090-9a668bdeb3cd.png)

Finally, just before the end of the level the game turns completely black, except for the interface. Up to this point, any input isn't registered but the music still plays and the timer still goes on (there is no crash):
![SG - Black Screen](https://user-images.githubusercontent.com/25135838/85903776-f1c78500-b7e5-11ea-962c-6294f84b2753.png)

### **Log File**
[Sonic Generations - Log File.zip](https://github.com/xenia-project/game-compatibility/files/4839417/Sonic.Generations.-.Log.File.zip)

### **Labels**
state-gameplay

**Comment by Calamity34** [comment]
nothing new on it yet?

**Comment by Machina74** [comment]
Experiencing the same issue as above with the latest canary. Get to about a 1/3 of any level as Modern Sonic (I transported my saves) and the screens goes black expect for the UI. Music still plays

**Comment by RagdolllFan2005** [comment]
soo I Think it may be possible to beat the game on canary with the glitches with cache

**Comment by murphyman236** [comment]
For me, whenever sonic boosts it makes the screen really blurry, do you know how to fix that?

**Comment by MilesTailsFan2005** [comment]
You can only press F5 To clear cache and this happens to sonic unleashed during cache loading it did its best to wait until xenia fixes the issue

**Comment by MilesTailsFan2005** [comment]
I Can Confirm Sonic Generations Is Completeable On Xenia 
![image](https://user-images.githubusercontent.com/52087744/232312321-94ecf513-1d11-489f-8631-2af430fe5d2c.png)

**Comment by Megasonic999** [comment]
As for me, I have the PC version of Sonic generations, and it runs completely from start to finish without any issues.

**Comment by minecrafter-124** [comment]
> As for me, I have the PC version of Sonic generations, and it runs completely from start to finish without any issues.

yes but it is about the fun of playing the xbox 360 version of the game on pc. if i where to just install the steam version of the game it would not be the same.

**Comment by namuHExists** [comment]
![Untitled](https://github.com/user-attachments/assets/e2e7d6b2-fbde-4967-ad9b-cbdc3044447b)
this game definitely works a lot better in xenia canary

**Comment by SteavenGamerYT** [comment]
![Image](https://github.com/user-attachments/assets/4d7f124f-caa2-4eef-91cc-cf2da8e776f0)
green hill
